plugins/modules/helm_template.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+
+module: helm_template
+
+short_description: Render chart templates
+
+author:
+  - Mike Graves (@gravesm)
+
+description:
+  - Render chart templates to an output directory or as text of concatenated yaml documents.
+
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  chart_ref:
+    description:
```

Comment:
Would this option be one of the listed items?  It might be more obvious to folks who are familiar with Helm but IMO we should say something like, "Chart reference with a repo prefix. Can be one of the following:" and then list the next 3 options.

Reply:
"Chart reference with a repo prefix" is actually one of the options (there are four). I can add a "for example" though, which should help clarify.